fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/wallet/ContentWalletFragment.java

```diff
@@ -51,38 +51,37 @@ public View onCreateView(
     mWalletContentBinding.setViewmodel(mHomeViewModel);
     mWalletContentBinding.setLifecycleOwner(requireActivity());
 
-    return mWalletContentBinding.getRoot();
-  }
-
-  @Override
-  public void onActivityCreated(@Nullable Bundle savedInstanceState) {
-    super.onActivityCreated(savedInstanceState);
-
     setupListAdapter();
     setupListUpdates();
 
     if (wallet.isSetUp()) {
-      mWalletContentBinding.logoutButton.setVisibility(View.VISIBLE);
-      mWalletContentBinding.logoutButton.setOnClickListener(
-          clicked -> {
-            if (logoutAlert != null && logoutAlert.isShowing()) {
-              logoutAlert.dismiss();
-            }
-            AlertDialog.Builder builder = new AlertDialog.Builder(requireActivity());
-            builder.setTitle(R.string.logout_title);
-            builder.setMessage(R.string.logout_message);
-            builder.setPositiveButton(
-                R.string.confirm,
-                (dialog, which) -> {
-                  mHomeViewModel.logoutWallet();
-                  WalletFragment.openWallet(
-                      getParentFragmentManager(), mHomeViewModel.isWalletSetUp());
-                });
-            builder.setNegativeButton(R.string.cancel, (dialog, which) -> dialog.dismiss());
-            logoutAlert = builder.create();
-            logoutAlert.show();
-          });
+      setupWalletContent();
```

Comment:
Just from a naming perspective, it seems weird to have the logic "if it's setup, set it up!" 
Can we distinguish between "setup" and "setup" ? 😅